controllers/nnf_workflow_controller.go

```diff
@@ -499,30 +508,27 @@ func (r *NnfWorkflowReconciler) generateDirectiveBreakdown(ctx context.Context,
 	// "#DW jobdw              type=gfs2   capacity=9GB name=thisIsGfs2"
 	// "#DW jobdw              type=lustre capacity=9TB name=thisIsLustre"
 	// "#DW create_persistent  type=lustre capacity=9TB name=thisIsPersistent"
+	// "#DW persistentdw       name=thisIsPersistent"
 
 	// #DW commands that require a dwDirectiveBreakdown because they create storage
```

Comment:
Comment is stale. persistentdw needs to specify which computes can be chosen for the WLM.